README.md

```diff
@@ -0,0 +1,158 @@
+## status-go-sdk
+This package provides an easy way to interact with [status](https://status.im) messaging system.
+This package allows you create or  application interact with [status](https://status.im) messaging system.
+
+## Motivation
+This library was initially created to have a low-dependency way to interact with status messaging system.
+
+## Build status
+
+[![Build Status](https://travis-ci.org/akashnimare/foco.svg?branch=master)](https://travis-ci.org/akashnimare/foco)
+
+## Features
+This project is under heavy development process, here is a list of supported features:
+
+**Status messaging basic interaction**
+- [x] Setup a new connection
+- [x] Close a specific connection
+- [x] Sign up on the platform
+- [x] Login as a specific account
+- [ ] Documented API for basic sdk interaction
+
+**Public channels interaction**
+- [x] Join public channels
+- [x] Publish messages on public channels
+- [x] Subscribe to public channels
+- [x] Unsubscribe from a public channel
+- [ ] Documented API for public channels interaction
+- [x] Working examples for public channel interaction
+
+**Contact management**
+- [ ] Invite new contact
+- [ ] Accept a contact request
+
+**1 to 1 messages interaction**
+- [ ] Send 1 to 1 conversation
+- [ ] Subscribe to 1 to 1 conversation
+- [ ] Unsubscribe from a 1 to 1 conversation
+- [ ] Documented API for 1 to 1 conversation
+- [ ] Working examples for 1 to 1 conversations
+
+**Private groups interaction**
+- [ ] Publish messages on private groups
+- [ ] Subscribe to private groups
+- [ ] Unsubscribe from a private groups
+- [ ] Documented API for private groups interaction
+- [ ] Working examples for private groups interaction
+
+## Code Example
+You'll find some code examples on the [examples folder](/examples)
+
+## Installation
+status-go-sdk is a lightweight dependency package, that means by now it does not have any external dependency. You can install it by running:
+```
+go get github.com/status-im/status-go
```

Comment:
Running the first basic example on line 81 requires `github.com/ethereum/go-ethereum/rpc` so maybe here you can add:
```
go get github.com/ethereum/go-ethereum/rpc
```
And also mention on line 52 that the only external dependencies are `status-go` and `go-ethereum` instead of saying that there are no external dependencies (although I understand that `status-go` can barely be considered 'external'

Reply:
yeah, its really confusing, i'm changing it to:

`status-go-sdk` is a lightweight dependency package, that means we try to avoid as much external dependencies as possible. 
You can install this package by running:
```
go get github.com/status-im/status-go-sdk
```
However, and to be able to run some examples you may also want to install `go-ethereum/rpc` with:
```
go get go get github.com/ethereum/go-ethereum/rpc
```